sympy/polys/ring_series.py

```diff
@@ -419,6 +418,61 @@ def rs_integrate(self, x):
         p1[e] = self[expv]/(expv[n] + 1)
     return p1
 
+def fun(p, f, *args):
+    """
+    Function of a multivariate series computed by substitution
+
+      p: multivariate series
+      f: method name or function
+      args[:-2] arguments of f, apart from the first one
+      args[-2] = iv: names of the series variables
+      args[-1] = prec: list of the precisions of the series variables
+
+    The case with f method name is used to compute tan and nth_root
+    of a multivariate series:
+
+      fun(p, tan, iv, prec)
+      compute _x.tan(iv, prec), then substitute _x with p
+
+      fun(p, nth_root, n, iv, prec)
+      compute (_x + p[0]).nth_root(n, '_x', prec)), then substitute _x
+      with p - p[0]
+
+    Examples
+    ========
+
+    >>> from sympy.polys.domains import QQ
+    >>> from sympy.polys.rings import ring
```

Comment:
You need to import `fun` here as well. See the Travis [failure](https://travis-ci.org/sympy/sympy/jobs/57463965):
```
_________________________ sympy.polys.ring_series.fun __________________________

File "/home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/sympy-0.7.6_git-py2.6.egg/sympy/polys/ring_series.py", line 448, in sympy.polys.ring_series.fun

Failed example:

fun(p, _tan1, 'x', 4)

Exception raised:

Traceback (most recent call last):

File "/opt/python/2.6.9/lib/python2.6/doctest.py", line 1253, in __run

compileflags, 1) in test.globs

File "<doctest sympy.polys.ring_series.fun[4]>", line 1, in <module>

fun(p, _tan1, 'x', 4)

NameError: name 'fun' is not defined

===== tests finished: 2909 passed, 1 failed, 14 skipped, in 479.21 seconds =====

DO *NOT* COMMIT!
```